config/config.js

```diff
@@ -4,3 +4,5 @@ window.BASE_URL = 'example.com'
 window.STREETS_TILE_LAYER_URL = 'https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
 window.LOGO_URL = 'https://placekitten.com/530/530'
 window.MAPBOX_ACCESS_TOKEN = ''
+window.AUTH0_DOMAIN = 'smartcolumbusos-demo.auth0.com'
+window.AUTH0_CLIENT_ID = 'sfe5fZzFXsv5gIRXz8V3zkR7iaZBMvL0'
```

Comment:
I assume this is a dummy client id?

Reply:
This is the ID for our 'demo' instance of AUTH0.  We will be overriding it in a future card.